app/Controller/SongsController.php

```diff
@@ -194,8 +194,8 @@ public function albums() {
                 'Song' => array(
                     'fields' => array('Song.id', 'Song.band', 'Song.album', 'Song.cover'),
                     'conditions' => $subQuery,
-                    'order' => 'Song.album',
-                    'limit' => 36
+                    'order' => ['Song.band', 'Song.album'],
+                    'limit' => 60
```

Comment:
Yes I tried and that didn't improve it :(
On Sep 12, 2016 4:50 PM, "Creak" <notifications@github.com> wrote:

> In app/Controller/SongsController.php
> <https://github.com/Sonerezh/sonerezh/pull/216#discussion_r78387433>:
>
> > @@ -194,8 +194,8 @@ public function albums() {
> >                  'Song' => array(
> >                      'fields' => array('Song.id', 'Song.band', 'Song.album', 'Song.cover'),
> >                      'conditions' => $subQuery,
> > -                    'order' => 'Song.album',
> > -                    'limit' => 36
> > +                    'order' => ['Song.band', 'Song.album'],
> > +                    'limit' => 60
>
> I'll try to reproduce it tonight and make a more thorough test.
>
> As for the sorting, as a temporary workaround you can put only
> 'Song.band', also have you tried using array() instead of []. I'm not
> sure PHP handles arrays the same when using these two syntax.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Sonerezh/sonerezh/pull/216/files/6d0ffa7da42730be1932b905db79c0f26265ebb9..a4bf12ef69f4c0b07f63dd1aaf3753dfcff5b36d#r78387433>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEjlgYFMQofqOOSsYUAetUxovNzlglbuks5qpWalgaJpZM4Jb51g>
> .
>